client/src/entities/player.rs

```diff
@@ -1,29 +1,60 @@
-use amethyst::core::math::Point2;
 use amethyst::core::Transform;
+use amethyst::core::transform::Parent;
 use amethyst::prelude::*;
+use amethyst::ecs::Entity;
 use log::info;
 
 use westiny_common::components::{Input, Health, Player, NetworkId};
 use crate::resources::SpriteResource;
 use westiny_common::resources::SpriteId;
 
-pub fn initialize_player<B: Builder>(builder: B,
-                         sprite_resource: &SpriteResource,
-                         network_id: NetworkId,
-                         start_pos: Point2<f32>
-                         ) {
-
-    let mut transform = Transform::default();
-    transform.set_translation_xyz(start_pos.x, start_pos.y, 0.0);
+pub fn create_hand_for_character<B: Builder>(
+    builder: B,
+    sprite_resource: &SpriteResource,
+    entity: Entity
+    ) -> Entity
+{
+    let mut hand_transform = Transform::default();
+    hand_transform.set_translation_xyz(-3., -6., 0.);
```

Comment:
Not has to be changed now just we should think about how could we calculate these offsets from sprite size

Reply:
I think that we cannot generate it automatically from the sprite size, but we might store this information separately from the code. (Eg in a ron file). To be padantic: this offset (where is Clint's shoulder) is partially an attribute of the parent, and partially comes from the sprite orientation.

This or that way, storing this information in a ron file would help us to faster align this hierarchies. (It is tedious to rebuild everything for one pixel change)